app/services/benefit_check_service.rb

```diff
@@ -0,0 +1,45 @@
+class BenefitCheckService
+  BENEFIT_CHECKER_NAMESPACE = 'https://lsc.gov.uk/benefitchecker/service/1.0/API_1.0_Check'.freeze
+
+  attr_reader :application
+
+  def initialize(application)
+    @application = application
+  end
+
+  def check_benefits
+    soap_client.call(:check, message: benefit_checker_params).body.dig(:benefit_checker_response)
```

Comment:
❓ 

* What happens when an exception occurs (connection issue or something unexpected)? (I'm assuming the user will see a 500 page).
* What happens when SOAP response is an error response (some details were wrong, the check could not be performed, etc)?

💭 Consider handling the SOAP exceptions/errors more smoothly so that the user sees the right outcome page (and leverage from Sentry to report any Exceptions raised).